scripts/README.md

```diff
@@ -94,14 +123,29 @@ scripts/gpu/setup_nvidia_gpu.sh
 ./setup_tool [options] setup_nvidia_gpu
 ```
 
-**NOTICE**: Shell scripts in `scripts/stargz`, `scripts/self-hosted-kind`, and `scripts/github_runner` **are not supported to be invoked in this way at present**.
+**NOTICE**: Shell scripts in `scripts/stargz`, `scripts/self-hosted-kind`, and `scripts/github_runner` **are not
+supported to be invoked in this way at present**.
+
+### 3.6 Kubernetes Control Plane High-Availability Mode
 
+For fault tolerance purposes, Kubernetes control plane components can be replicated. This can be done by combining
```

Comment:
Still better have a description of what and how options are used (specifically, REGULAR, which appears throughout the docs).